subprojects/core/src/main/java/org/gradle/execution/plan/TaskInAnotherBuild.java

```diff
@@ -139,6 +139,11 @@ public boolean isSuccessful() {
         return state == IncludedBuildTaskResource.State.Success;
     }
 
+    @Override
+    public boolean isVerificationFailure() {
+        return false;
```

Comment:
That's correct; for version 0 we won't support aggregating across repository boundaries. 

@big-guy are you OK with this?

Reply:
I agree that composite builds should probably work, but for version 0, this isn't important.

We should have a test at least that demonstrates the current behavior. I'd like to wait to add that coverage after we have some more of the aggregation work merged since it's very finicky to setup an integration test that uses composite builds and could do something with verification failures.

I added an issue to add this soon #18916

Reply:
Are included builds in scope for the SOW?  I thought I remember a decision that they weren't, but didn't see another about them in the Verification Failures Spec.

Reply:
The initial trouble of implementing here is that `getNodeFailure()` throws an exception. When we get to supporting composite builds (and I agree, we should), we will have another look at this.

Reply:
👍

Reply:
Can you also open an issue for all this stuff working through artifact transforms?

Reply:
https://github.com/gradle/gradle/issues/18932